catroidTest/src/org/catrobat/catroid/test/content/actions/PreviousLookActionTest.java

```diff
@@ -0,0 +1,183 @@
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.test.content.actions;
+
+import android.test.InstrumentationTestCase;
+
+import com.badlogic.gdx.scenes.scene2d.Action;
+
+import org.catrobat.catroid.ProjectManager;
+import org.catrobat.catroid.common.Constants;
+import org.catrobat.catroid.common.LookData;
+import org.catrobat.catroid.common.ScreenValues;
+import org.catrobat.catroid.content.Project;
+import org.catrobat.catroid.content.Sprite;
+import org.catrobat.catroid.content.actions.NextLookAction;
+import org.catrobat.catroid.content.actions.PreviousLookAction;
+import org.catrobat.catroid.content.actions.SetLookAction;
+import org.catrobat.catroid.io.StorageHandler;
+import org.catrobat.catroid.test.R;
+import org.catrobat.catroid.test.utils.TestUtils;
+import org.catrobat.catroid.uitest.util.UiTestUtils;
+import org.catrobat.catroid.utils.UtilFile;
+
+import java.io.File;
+
+public class PreviousLookActionTest extends InstrumentationTestCase {
+
+	private static final int IMAGE_FILE_ID = R.raw.icon;
+	private File testImage;
+	private String projectName = UiTestUtils.DEFAULT_TEST_PROJECT_NAME;
+
+	@Override
+	protected void setUp() throws Exception {
+
+		File projectFile = new File(Constants.DEFAULT_ROOT + "/" + projectName);
+
+		if (projectFile.exists()) {
+			UtilFile.deleteDirectory(projectFile);
+		}
+
+		UiTestUtils.createEmptyProject();
+		Project project = new Project(getInstrumentation().getTargetContext(), projectName);
+		StorageHandler.getInstance().saveProject(project);
+		ProjectManager.getInstance().setProject(project);
+
+		testImage = TestUtils.saveFileToProject(projectName, "testImage.png", IMAGE_FILE_ID, getInstrumentation()
+				.getContext(), TestUtils.TYPE_IMAGE_FILE);
+
+		ScreenValues.SCREEN_HEIGHT = 200;
+		ScreenValues.SCREEN_WIDTH = 200;
+	}
+
+	@Override
+	protected void tearDown() throws Exception {
+		File projectFile = new File(Constants.DEFAULT_ROOT + "/" + projectName);
+
+		if (projectFile.exists()) {
+			UtilFile.deleteDirectory(projectFile);
+		}
+		if (testImage != null && testImage.exists()) {
+			testImage.delete();
+		}
+		super.tearDown();
+	}
```

Comment:
> you have forgotten to add the brick's xstream aliases to the StorageHandler class

Ticket [CAT-1711](https://jira.catrob.at/browse/CAT-1711) addresses this "problem". However, each approach has it pros and cons. Defining aliases per annotation would have been less error-prone in this case. Since new bricks are mostly c&ped, the annotation will be c&ped as well.